### PR TITLE
[NEW] A clearer, more readable return statement

### PR DESCRIPTION
**Pull request by sidiousvic**
.filter() doing the odd value filtering, .reduce() doing the reducing.
In any case, the current solution's use of .reverse() is unnecessary.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/index.md [hunk 1]
```diff
@@ -90,9 +90,7 @@ As you get the next odd one, don't forget to add it to a global variable that ca
 
         // Sum only the odd numbers and return the value
         // First, reverse the array to avoid starting acc with the first/greater number when it's even
```
Comment (Dhirendra24) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/index.md]: Please update the comment & `Code Explanation` section as well.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/index.md [hunk 2]
```diff
@@ -88,11 +88,8 @@ As you get the next odd one, don't forget to add it to a global variable that ca
             arrFib.unshift(nextFib);
         }
 
-        // Sum only the odd numbers and return the value
-        // First, reverse the array to avoid starting acc with the first/greater number when it's even
-        return arrFib.reverse().reduce((acc, curr) => {
-            return acc + curr * (curr % 2);
-        }, 0);
+        // We return the array, filtering and summing (reducing) odd numbers at once.
```
Comment (ojeytonwilliams) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/index.md]:
```suggestion
        // We filter the array to get the odd numbers and reduce them to get their sum.
```
I wasn't sure about the original wording as it sounded like we were returning the array itself.

**Review by ojeytonwilliams — COMMENTED** [review]
Other than that one wording quibble, it looks good to me!

**Comment by RandellDawson** [comment]
@ojeytonwilliams This PR has gone stale.

**Comment by ojeytonwilliams** [comment]
@sidiousvic Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Pull request merged**